### ISSUE TITLE
Java problems after fresh Mint 19 install (Minecraft won't start)

### ISSUE THREAD

**Issue by pekkaklarck** [linuxmint/cinnamon#7821]
```
 * Fresh Linux Mint 19 Cinnamon install
 * ThinkPad T420
 * 64 bit
 ```

**Issue**

Installed laptop with our kid mainly for him to play Minecraft on it. When I noticed Minecraft being listed in as a recommended program during the installation thought everything must go fine. The first thing we did after the Mint installation (and upgrading of all packages) was installing Minecraft, but although there were no reported errors the program didn't start. Tried both clicking button in the installation dialog itself and selecting from the start menu but nothing happened, there weren't even any error. Only after running `minecraft` on the command line I got this error:

    Exception in thread "main" java.awt.AWTError: Assistive Technology not found: org.GNOME.Accessibility.AtkWrapper

A little googling revealed that this is somehow related to included headless Java installation but `dpkg -l | grep openjdk` told also normal JRE was installed. I also found it's possible to [workaround this problem by disabling accessibility properties](https://askubuntu.com/questions/695560/assistive-technology-not-found-awterror/723503#723503). After doing that the program starts, but I'm a little worried what side-effects that change possibly has.

**Steps to reproduce**

Already explained above but here's to rephrase:

- Install Linux Mint 19 Cinnamon
- Update all installed packages
- Install Minecraft
- Try to run Minecraft

**Expected behaviour**

I'd expect Minecraft to start, especially when it was shown as a recommended program. Most likely the problem has nothing to do with Minecraft itself, but instead this is a Java configuration issues.

**Other information**

Linux Mint 19 Cinnamon looks great! Need to install it to my own laptop over an older Mint installation as well. Thanks for the hard work for the whole team!

Also, sorry if this was a wrong issue tracker to submit this issue to. It wasn't exactly clear which one to use.

**Comment by collinss** [comment]
Yeah, it wasn't working for me either. My problem was a little different though. The launcher kept crashing on me. I worked around it by using the official JRE from Oracle. Apparently only some people are affected though. I'm not sure why.

**Comment by mtwebster** [comment]
Are these 2 packages installed?

```
libatk-wrapper-java
libatk-wrapper-java-jni
```
They should already be installed though (I think)

Regardless, disabling that as you did is perfectly fine.  Unless you need to use assistive technologies (such as the screen reader or other accessibility features) you won't lose anything.
